src/TogglePassword/assets/src/controller.ts

```diff
@@ -57,7 +55,7 @@ export default class extends Controller<HTMLInputElement> {
         button.classList.add(...this.buttonClassesValue);
         button.setAttribute('tabindex', '-1');
         button.addEventListener('click', this.toggle.bind(this));
-        button.innerHTML = this.visibleIcon + ' ' + this.visibleLabelValue;
+        button.innerHTML = `${this.visibleIcon} ${this.visibleLabelValue}`;
```

Comment:
Is this worth the change ?

Reply:
Yes, string concatenation with `+` is a mistake